### PR TITLE
Api version fix

### PR DESCRIPTION
**Pull request by llamashoes**
This gets rid of the deprecation warning referenced in #9.

### CODE REVIEW

**Comment by vassilvk** [comment]
@llamashoes, thanks!
Please bump the version of the helm chart to 0.2.2 (see failed github action check).

@desaintmartin, please review the PR.

File: helm-chart/kubemod/crds/crd-kubemod.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```
Comment (desaintmartin) [inline helm-chart/kubemod/crds/crd-kubemod.yaml:2]: We should reflect this on the minimum version of kubernetes in chart.yaml.

Comment (llamashoes) [reply helm-chart/kubemod/crds/crd-kubemod.yaml:2]: Good idea... will make the change

Comment (llamashoes) [reply helm-chart/kubemod/crds/crd-kubemod.yaml:2]: I have pushed this. Sorry for the delay.

Comment (vassilvk) [reply helm-chart/kubemod/crds/crd-kubemod.yaml:2]: Thank you @llamashoes!

I think there's only one thing left - @desaintmartin's suggestion to add `kubeVersion` field to Chart.yaml and set it to greater than or equal to Kubernetes 1.16, where `v1beta1` CRDs were deprecated.

Please see https://helm.sh/docs/topics/charts/#the-kubeversion-field

Comment (llamashoes) [reply helm-chart/kubemod/crds/crd-kubemod.yaml:2]: Sweet! Looking now...

Comment (vassilvk) [reply helm-chart/kubemod/crds/crd-kubemod.yaml:2]: Thanks @llamashoes !

**Pull request merged**